lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -335,6 +335,8 @@ def create_tar(cv, cvv, repositories, json)
 
             `tar cvfh #{repo_tar} #{repo_dirs.join(" ")}`
           end
+        else
+          puts "Ensure the content view '#{cvv['name']}' has at least one repository."
```

Comment:
Hmm... I just realized another issue. The test that was failing was for composite content view export (in which repositories is nil). This message doesn't make sense in that context.

Reply:
IMO you can fix the composite issue by checking `elsif !cv['composite']`